floodviz/static/geojson/reference.json

```diff
@@ -0,0 +1,31 @@
+{
```

Comment:
See previous comment. This is configuration information and should not be part of the repo.